x-pack/test_serverless/functional/test_suites/search/landing_page.ts

```diff
@@ -19,7 +19,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
   describe('landing page', function () {
     before(async () => {
-      await pageObjects.svlCommonPage.loginWithRole('viewer');
+      await pageObjects.svlCommonPage.loginWithPrivilegedRole();
```

Comment:
Why do we change `viewer` role to a privileged role in this test?

Reply:
https://github.com/elastic/kibana/pull/188128/commits/56b5152dbd7312b38b946cffeca1ee8389462303